scout/server/blueprints/public/templates/public/index.html

```diff
@@ -66,3 +66,31 @@ <h1 class="display-4">Scout</h1>
   </main>
 </div>
 {% endblock %}
+
+{% block scripts %}
+{{ super() }}
+  <script type="text/javascript">
+    const sendBrowserInfo = () => {
+      console.log('Calling Netlify')
+      if (window.location.hostname.includes('scilifelab.se')) {
+          fetch('https://user-stats.netlify.app/.netlify/functions/add-browser', {
```

Comment:
Does this document apply for this feature? https://www.netlify.com/gdpr-ccpa/ 

I think should be discussed with the users before merging..

Reply:
Ok no, this is for when you use their software to deploy your app!